scribble-lib/scribble/latex-render.rkt

```diff
@@ -21,6 +21,7 @@
 (define disable-images (make-parameter #f))
 (define escape-brackets (make-parameter #f))
 (define suppress-newline-content (make-parameter #f))
+(define disable-hyperref (make-parameter #f))
```

Comment:
I'm not sure what the goal of this parameter is.

If it's trying to decide whether to print `\hyperref{....}` or just the contents `....`, I think it should just always do `hyperref` (or a Scribble macro)

Reply:
Putting a `\hyperref` in things like `\section` causes the table of contents to go bad (and then Latex errors out), so the internal parameter is meant to work around problems like that.